commons/zenoh-protocol/src/core/endpoint.rs

```diff
@@ -880,4 +880,8 @@ fn endpoints() {
     assert_eq!(i.next(), Some("224.0.0.2"));
     assert_eq!(i.next(), Some("224.0.0.3"));
     assert_eq!(i.next(), None);
+
+    let endpoint = EndPoint::from_str("udp/127.0.0.1:7447#bind=224.0.0.1:8080").unwrap();
```

Comment:
Nice but not necessary since in `endpoint.rs` `key=value` is purely syntactic and does not carry any semantics.